documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,382 @@ keywords:
+Endpoints must be valid URLs. Invalid URLs will cause an error to be thrown in JavaScript.
+:::
+
+:::tip
+You can can add custom properties to your service by setting `properties` in both Rust and JavaScript.
+:::
+
+Now The updated Document with the new service looks as follows.
+```json
+{
+   "doc":{
+      "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+      "verificationMethod":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-next-key",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z2Zthec5siTfxCjPwZUHGDGybKNy9oc3ZYeftvEE2nEL3"
+         }
+      ],
+      "capabilityDelegation":[
+         "did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-next-key"
+      ],
+      "capabilityInvocation":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#sign-0",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z5k7vzMVuXXj8MJDcfzP2owvc8xKBA6BBsAkFf1GSNu2X"
+         },
+         "did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-next-key"
+      ],
+      "service":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-service-1",
+            "type":"MyCustomService",
+            "serviceEndpoint":"https://example.com/"
+         }
+      ]
+   },
+   "meta":{
+      "created":"2022-04-13T09:27:48Z",
+      "updated":"2022-04-13T09:28:34Z"
+   }
+}
+```
+
+### 5. Removing a Verification Method
+
+Verification Methods and/or their Relationships can be removed from the DID Document. The Following code removes the Verification Method that we created previously.
+
+<Tabs>
+<TabItem value="Rust" label="Rust">
+
+```rust
+account
+    .update_identity()
+    .delete_method()
+    .fragment("my-next-key")
+    .apply()
+    .await?;
+```
+
+</TabItem>
+<TabItem value="node" label="Node.js">
+
+```js
+await account.deleteMethod({fragment: "my-next-key"})
+```
+</TabItem>
+</Tabs>
+
+```json
+{
+   "doc":{
+      "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+      "capabilityInvocation":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#sign-0",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z5k7vzMVuXXj8MJDcfzP2owvc8xKBA6BBsAkFf1GSNu2X"
+         }
+      ],
+      "service":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-service-1",
+            "type":"MyCustomService",
+            "serviceEndpoint":"https://example.com/"
+         }
+      ]
+   },
+   "meta":{
+      "created":"2022-04-13T09:27:48Z",
+      "updated":"2022-04-13T09:29:03Z"
+   }
+}
+```
+
+:::note
```

Comment:
As before, transform all notes and tips into a good paragraph explaining it and only use it too really highlight something that is crucial.